uint32 id
geometry_msgs/PoseWithCovariance pose